Bio/motifs/jaspar/db.py

```diff
@@ -81,8 +81,7 @@
 
 
 class JASPAR5(object):
-    """
-    Class representing a JASPAR5 DB. The methods within are loosely based
+    """Class representing a JASPAR5 DB. The methods within are loosely based
```

Comment:
I'd use just ``Class representing a JASPAR5 DB.`` or ``Class representing a JASPAR5 database.`` as the one line summary, the rest can stay in the main body of the docstring.